book/src/editor.md

```diff
@@ -31,6 +31,7 @@
 | `cursorline` | Highlight all lines with a cursor | `false` |
 | `cursorcolumn` | Highlight all columns with a cursor | `false` |
 | `gutters` | Gutters to display: Available are `diagnostics` and `diff` and `line-numbers` and `spacer`, note that `diagnostics` also includes other features like breakpoints, 1-width padding will be inserted if gutters is non-empty | `["diagnostics", "spacer", "line-numbers", "spacer", "diff"]` |
+| `command-git-ignore` | Whether to read `.gitignore` when listing paths in the command prompt | `true` |
```

Comment:
shouldn't this default to false to avoid changing behavior?

Reply:
The current behavior is to read `.gitignore`, so the default should be true. I named it this way to follow the pattern of the existing `editor.file-picker.git-ignore` option.